docs/production/authentication-methods.md

```diff
@@ -439,6 +439,37 @@ details.
 
 [upstream-ldap-groups]: https://django-auth-ldap.readthedocs.io/en/latest/groups.html#limiting-access
 
+### Restricting access to specific organizations
+
+If you're hosting multiple Zulip organizations, you can restrict which
+organizations can be accessed through LDAP by a specific user.
+Setting `org_membership` in `AUTH_LDAP_USER_ATTR_MAP` to a valid name of
+a LDAP argument which value matches the subdomain of the organization the
+user should be allowed to access.
+
+For the root subdomain, `www` in the list will work, or any other of
+`settings.ROOT_SUBDOMAIN_ALIASES`.
+
+For example, with `org_membership` set to `department`, a user with
+the following attributes will have access to the root and `engineering` subdomains:
+```
+...
```

Comment:
why do we have double ` ``` `?

Reply:
I do not really understand your question.
Line 456 are not ```, but dots to indicate that there are other attributes, that do not matter to us.

Reply:
Ah, github UI make the dots look like ````` to me :slightly_smiling_face: One more question, as I'm not that familiar with ldap syntax - is the example given here  how multivalued attributes are specificied? By having multiple lines with the attribute, specifying the values?

Reply:
I sadly could not find a source that shows that it is done like this.
Only thing I could find is [this Stackoverflow Awnser](https://stackoverflow.com/a/43155092/8314475).